### DISCUSSION TITLE
The application has been passed, but the student welfare has not been issued, and the specific time is not clear.

### DISCUSSION THREAD

**Discussion by 42095308** [community/community#70702] [GitHub Education] [answered]
### Hi everyone,

I approved the application on October 17th, but_ now it has been four days, and my education benefits have not been paid.
![微信图片_20231021133121](https://github.com/community/community/assets/107547694/22391527-85cd-4f31-918b-1e1b63b66c9e)
#

**Comment by 42095308** [comment, marked as answer]
Can anyone answer my question?

**Reply by aire233** [reply]
I have got the authentication at Oct 19th, but also did not get the benefits.

**Reply by BraveDevelopment** [reply]
Same

**Reply by jzlhk** [reply]
same here

**Reply by nickolasdeluca** [reply]
So... did you managed to get the benefits? I'm still waiting on mine and nothing...

**Reply by BraveDevelopment** [reply]
Yeah I got it. Took 5 days but I got it. Five days AFTER my application was approved!

**Reply by ankitkat042** [reply]
@42095308, what is the answer? What is the maximum time one should wait to receive the student program benefits once the application has been approved?

**Reply by wolf1994722** [reply]
same here

**Reply by ankitkat042** [reply]
Yes, for me It took 3 days. Got approved on 10th Jan and received benefits on 13th Jan

**Comment by Feisal2023** [comment]
Hi, Feisal Issack Adan! You were last verified as a student on Oct 26, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits. still i can't access my education benefits yet also three day after my approval ???
